modules/storage.js

```diff
@@ -1,12 +1,7 @@
 export default class Storage {
   static getBooks() {
-    let books;
-    if (localStorage.getItem('books') === null) {
-      books = [];
-    } else {
-      books = JSON.parse(localStorage.getItem('books'));//
-    }
-    return books;
+    const books = JSON.parse(localStorage.getItem('books')) || [];
```

Comment:
- [ ] After I added a book and went to the book index page I got an error and no books showed up. Kindly fix it
![image](https://user-images.githubusercontent.com/62705947/220159904-2b0a1972-69db-40b0-b6c6-b80d5f5722e2.png)

Reply:
Hey I can't seem to recreate the error that you are facing